shell/platform/windows/public/flutter_windows.h

```diff
@@ -201,6 +202,11 @@ FLUTTER_EXPORT void FlutterDesktopEngineSetNextFrameCallback(
 // Return backing HWND for manipulation in host application.
 FLUTTER_EXPORT HWND FlutterDesktopViewGetHWND(FlutterDesktopViewRef view);
 
+// Attempts to return the DXGI adapter used for rendering.
```

Comment:
```suggestion
// Returns the DXGI adapter used for rendering, if available.
```

Reply:
Please add a comment `// Returns nullptr when...` that emphasises that there are cases in which the user should expect nullptr.

Initially I'd assumed this would be the case when software rendering is enabled, but it seems like software adapters [exist](https://docs.microsoft.com/en-us/windows/win32/api/dxgi/nf-dxgi-idxgifactory-createsoftwareadapter). If the only case where we expect nullptr is in case of error, then we could just state `Returns nullptr in case of error`.